lib/workers/repository/update/pr/index.ts

```diff
@@ -335,6 +357,8 @@ export async function ensurePr(
         });
         logger.info({ pr: existingPr.number, prTitle }, `PR updated`);
       }
+      // update prCache last edit time
+      setPrCache(branchName, prFingerprint);
```

Comment:
Update even if dry run? Shouldn't this be moved inside the above `else`?

Reply:
I added it inside first but in review stage I shifted all `setX` above return statements and didn't think twice :(